postgres/src/main/scala/zio/sql/postgresql/PostgresModule.scala

```diff
@@ -115,10 +117,8 @@ trait PostgresModule extends Jdbc { self =>
     private[zio] def renderLit[A, B](lit: self.Expr.Literal[_])(implicit render: Renderer): Unit = {
       import TypeTag._
       lit.typeTag match {
-        case tt @ TByteArray      => render(tt.cast(lit.value))                     // todo still broken
-        //something like? render(tt.cast(lit.value).map("\\\\%03o" format _).mkString("E\'", "", "\'"))
-        case tt @ TChar           =>
-          render("'", tt.cast(lit.value), "'") //todo is this the same as a string? fix escaping
+        case tt @ TByteArray      => render(tt.cast(lit.value).map("""\%03o""" format _).mkString("E\'", "", "\'"))
```

Comment:
let's just cast here where it's not inferred correctly so we can keep covariance:
```scala
case TByteArray => render(lit.value.asInstanceOf[Chunk[Byte]].map("""\%03o""" format _).mkString("E\'", "", "\'")) //todo fix `cast` infers correctly but map doesn't work for some reason
```